tests/director/tests/application_templates_subscription_test.go

```diff
@@ -451,13 +433,135 @@ func TestSubscriptionApplicationTemplateFlow(baseT *testing.T) {
 	})
 }
 
-func createSubscription(t *testing.T, ctx context.Context, httpClient *http.Client, appTmpl graphql.ApplicationTemplate, apiPath, subscriptionToken, subscriptionConsumerTenantID, subscriptionConsumerSubaccountID, subscriptionProviderSubaccountID, subscriptionProviderAppNameValue string, expectedToPass, unsubscribeFirst bool) {
-	subscribeReq := buildSubscriptionRequest(t, ctx, subscriptionProviderSubaccountID, subscriptionProviderAppNameValue)
+func TestSubscriptionApplicationTemplateFlowWhenIndirectDependency(baseT *testing.T) {
```

Comment:
```suggestion
func TestSubscriptionApplicationTemplateFlowWithIndirectDependency(baseT *testing.T) {
```
?
If you decide to change it do not forget to update the overrides in the config repo